crates/common/tedge_utils/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub mod file;
 pub mod fs;
 pub mod paths;
 pub mod signals;
+pub mod size_threshold;
```

Comment:
[Note] This is typically the kind of code I see no point to abstract. This is just a size comparison but adds a coupling to the `mqtt_channel` crate.